projects/bp-gallery/src/hooks/get-pictures.hook.ts

```diff
@@ -52,8 +57,9 @@ const useGetPictures = (
       },
       sortBy,
     },
+    fetchPolicy,
     notifyOnNetworkStatusChange: true,
-    skip: isAllSearchActive,
+    skip: isAllSearchActive || type !== PictureOverviewType.CUSTOM,
```

Comment:
the skip parameters is currently bugged. We'd have to change this in multiple places, so I created a new issue: https://github.com/hpi-swa-lab/BP2021RH1/issues/528.